File: bounded-contexts/dotnet/MyShop.Entities/Dvd.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MyShop.Entities
{
    public class Dvd : IItem
    {
        public ItemId ItemId => default(ItemId);
        public string ean => default(string);
        public double Weight => default(double);
    }
}

File: bounded-contexts/java/src/main/java/p1/myshop/catalog/entities/Dvd.java
package p1.myshop.catalog.entities;


public class Dvd implements Product {
    private ProductId productId;
    private String ean;

    public Dvd(ProductId productId, String ean) {
        this.productId = productId;
        this.ean = ean;
    }

    @Override
    public ProductId itemId() {
        return null;
    }

    @Override
    public double weight() {
        return 0;
    }

    public String getEan() {
        return ean;
    }
}
